src/nl/hannahsten/texifyidea/action/wizard/graphic/InsertGraphicWizardAction.kt

```diff
@@ -49,6 +49,28 @@ class InsertGraphicWizardAction(private val initialFile: File? = null) : AnActio
         editor.editor.insertGraphic(project, graphicData, indent)
     }
 
+    /**
+     * Opens and handles the graphic insertion wizard.
+     */
+    fun getActionString(file: VirtualFile, project: Project): String {
```

Comment:
Here the method is named `getActionString`, but it is not clear to me what an action string is exactly?

Reply:
This is a misnomer, it should be `getGraphicString`. it will return the string to insert into the file when pasting an image